<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1173</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1153</width>
        <height>600</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="opisLabel">
           <property name="text">
            <string>Izaberite postavke za spajanje</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="layoutZaGraf">
           <property name="margin">
            <number>0</number>
           </property>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="startButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="stopButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="spremiButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Spremi</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clearButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Reset</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="postavkeButton">
             <property name="text">
              <string>Postavke</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="sampleLabel">
             <property name="text">
              <string>Period uzorkovanja (s) :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="sampleCombo">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>10</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>15</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>20</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>30</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>60</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="rasponLabel">
             <property name="text">
              <string>Vremenski raspon grafa (min) :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="rasponCombo">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>10</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>15</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>30</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>60</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>90</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>120</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Raspon Y osi (min) :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBoxMinY">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="minimum">
              <double>-1000000.000000000000000</double>
             </property>
             <property name="maximum">
              <double>1000000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Raspon Y osi (max) :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="doubleSpinBoxMaxY">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="minimum">
              <double>-1000000.000000000000000</double>
             </property>
             <property name="maximum">
              <double>1000000.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
